common/lib/capa/capa/templates/chemicalequationinput.html

```diff
@@ -11,7 +11,7 @@
              % endif
       />
 
-      <p class="status">
+      <p class="indicator-container">
```

Comment:
While testing jsinput, I noticed that the screen reader "Question 1: correct" text was getting duplicated. This is because the JavaScript method looks for things with class "status", and some of these older/more obscure problem types had a p-tag with class of status around the new status_span.html file, which itself contains an element with class "status".

To fix this, changing the outer classname to "indicator-container", which is consistent with the more standard problem types.

Instances of this problem type throw a code jail exception on devstack/sandbox/staging. On production, I was able to submit one of these problem types, and I changed the class from "status" to "indicator-container" in the dev tools and verified that it does render correctly.
https://edge.edx.org/courses/edX/DemoX/Demonstration_Course/courseware/f793d4ead0054462a8b53a098581b693/basic_questions/3?activate_block_id=i4x://edX/DemoX/vertical/vertical_1fef54c2b23b